src/flick/show/utils.py

```diff
@@ -33,46 +36,65 @@
 """
 
 
-def get_movie_from_DBinfo(info):
+def get_movie_from_DBinfo(info, credit):
     """
     get a flick movie object similar by parsing the
     information returned by movieDB
     """
     genres = info.get("genres")
-    tags = [genre.get("name") for genre in genres] if genres else []
+    crew = credit.get("crew")
```

Comment:
rename to "credits"

Reply:
hmm i think credits is a constant in python